dev/tasks/tasks.yml

```diff
@@ -929,6 +930,13 @@ tasks:
       - plasma-glib-libs-{no_rc_version}-1.el7.x86_64.rpm
       - plasma-libs-{no_rc_version}-1.el7.x86_64.rpm
 
+  ############################## Homebrew Tasks ################################
+
+  homebrew-formula:
+    ci: travis
+    platform: osx
+    template: homebrew-formulae/travis.osx.yml
+
```

Comment:
Could you add

```yaml
params:
  formula: apache-arrow
```

and refer it by `{{ formula }}` in `dev/tasks/homebrew-formulae/travis.osx.yml`?

We may add `apache-arrow-glib.rb` later.